doc/reference/reference_lua/net_box.rst

```diff
@@ -205,6 +205,18 @@ Below is a list of all ``net.box`` functions.
       also use ``conn`` database methods (in this case, only the binary protocol is supported).
       Deprecation notice: ``console = true`` is deprecated, users should use
       :ref:`console.connect() <console-connect>` instead.
+      
+    * ``required_protocol_version``: depending on the value of the option, the connection requires the 
+       minimum version of the IPROTO protocol supported by the server. If the server version is lower 
+       than specified, the connection will fail with an error message. 
+       With ``required_protocol_version = 1`` all connections fail where the server protocol is lower than ``1``.
+      
+    * ``required_protocol_features``: depending on the value of the option, the connection requires 
+       the specified IPROTO protocol features supported by the server. If the server does not have
+       the specified features, the connection will fail with an error message.
+       With ``required_protocol_features = {'transactions'}`` all connections fail where the server 
+       has ``transaction: false``.
```

Comment:
It'd be nice to enumerate all available protocol features here, explain what each feature means and what protocol version it first appeared in. Maybe, refer to the [`IPROTO_ID`](https://www.tarantool.io/en/doc/latest/dev_guide/internals/box_protocol/#iproto-id-0x49) documentation section. Here's the list of features:
net.box feature name | IPROTO feature id | version
-|-|-
`streams` | `IPROTO_FEATURE_STREAMS` | version >= 1
`transactions` | `IPROTO_FEATURE_TRANSACTIONS` | version >= 1
`error_extension` | `IPROTO_FEATURE_ERROR_EXTENSION` | version >= 2
`watchers` | `IPROTO_FEATURE_WATCHERS` | version >= 3

Reply:
Added.